### PR TITLE
GUI-2831 - ELB Wizard directive

### PR DESCRIPTION
**Pull request by coyote240**
https://eucalyptus.atlassian.net/browse/GUI-2831

### CODE REVIEW

File: eucaconsole/static/js/pages/elb-wizard/wizard.js [hunk 1]
```diff
@@ -101,6 +117,9 @@ angular.module('ELBWizard', [
         },
 
         displaySummary: function(step) {
+            if(!this.nav) {
+                return;
```
Comment (dkavanagh) [inline eucaconsole/static/js/pages/elb-wizard/wizard.js]: Thinking this should return explicit truth value. I guess undefined == false, but I like explicitness.

File: eucaconsole/templates/elbs/wizard/main.pt [hunk 2]
```diff
@@ -32,9 +32,9 @@
         </h3>
 
         <div class="large-8 columns">
-            <div class="wizard">
+            <div class="wizard" ng-app="ELBWizard" elb-wizard="" vpc-enabled="${ is_vpc_supported }" cloud-type="${ cloud_type }">
```
Comment (dkavanagh) [inline eucaconsole/templates/elbs/wizard/main.pt]: ng-app is defined below as well. Is that by accident?

File: eucaconsole/static/js/pages/elb-wizard/wizard.js (hunk 1 above)
Comment (coyote240) [inline eucaconsole/static/js/pages/elb-wizard/wizard.js]: I tend to agree in Python, less so in JS, in general.  Note this next time you write a function that returns like ```return this.value && this.oldValue``` which will return this.oldValue.  That being said, I don't mind adding ```false.```

Comment (coyote240) [inline eucaconsole/static/js/pages/elb-wizard/wizard.js]: Oh.  I can use line 123 of this very file as an example.

**Pull request merged**